website/docs/relay_proxy/configure_relay_proxy.md

```diff
@@ -31,6 +31,7 @@ If you want to replace a nested fields, please use `_` to separate each field _(
 | `startWithRetrieverError` | boolean                   | `false`     | By default the **relay proxy** will crash if he is not able to retrieve the flags from the configuration.<br/>If you don't want your relay proxy to crash, you can set `startWithRetrieverError` to true. Until the flag is retrievable the relay proxy will only answer with default values. |
 | `exporter`                | [exporter](#exporter)     | **none**    | Exporter is the configuration on how to export data.                                                                                                                                                                                                                                          |
 | `notifier`                | [notifier](#notifier)     | **none**    | Notifiers is the configuration on where to notify a flag change.                                                                                                                                                                                                                              |
+| `apiKeys` | map[string]bool | **none** | List of authorized API keys. Each request will need to provide one of authorized key inside `Authorization` header with format `Bearer <api-key>`. There will be no authorization when this config is not configured.<br /><br />_Note: there will be no authorization when this config is not set._ |
```

Comment:
You mention twice this, maybe we should remove one.
> There will be no authorization when this config is not configured.

Reply:
these 2 docs looks similar
https://github.com/thomaspoignant/go-feature-flag/blob/HEAD/cmd/relayproxy/docs/configuration.md
https://github.com/thomaspoignant/go-feature-flag/blob/main/website/docs/relay_proxy/configure_relay_proxy.md
maybe can just keep one

Reply:
You are right.
I guess we could delete the one in `cmd/relayproxy/docs` since it is not really used and keep only the one that finish on the website.